### DISCUSSION TITLE
Query Timeout Issue: Seeking Input on Superset Dashboard Integration

### DISCUSSION THREAD

**Discussion by vinodvpillai** [betodealmeida/shillelagh#447] [Q&A]
Hi @betodealmeida I'm facing an issue and I need your input. I have read your article at https://preset.io/blog/accessing-apis-with-superset/ and followed the instructions to install Superset (Version 3) locally, along with the Shillelagh (Version 1.2.19) library as mentioned in the article.

Currently, I am encountering a challenge with the Dashboard functionality. I am able to create the necessary Dataset and visualize the data in chart format under the Chart section. However, when I try to integrate the chart into the Dashboard, it fails to display and eventually returns a timeout error.

Upon review, I noticed that the query generated in the chart section and the query generated in the Dashboard section are identical.

Could you please suggest a solution to this issue, or provide any relevant articles that may help?

Thank you.

**Comment by betodealmeida** [comment]
It's hard to say without more details. What APIs are you accessing? Could it be that the API is being throttled?
